source/game/system/map/MapdataCheckPath.hh

```diff
@@ -21,6 +22,17 @@ public:
     MapdataCheckPath(const SData *data);
     void read(EGG::Stream &stream);
 
+    void findDepth(s8 depth, const MapdataCheckPathAccessor &accessor);
+    bool isPointInPath(u16 checkpointId) const;
+
+    /// @beginGetters
+    [[nodiscard]] u8 start() const;
```

Comment:
Is this used? I'm on mobile and can't easily tell

Reply:
[yes:](https://github.com/KooShnoo/Kinoko/blob/a9d0703a577a13b0caf46ab874c1c1ae671f1a9c/source/game/system/map/MapdataCheckPoint.cc#L72-L74) `m_nextPoints[i].checkpoint = accessor.get(next->start());`